aamt_parse2.pm

```diff
@@ -296,7 +544,7 @@ sub pars_CreateReadinessReport
         $req->content( $json_text );
 
         my $publishSuccess = FALSE;
-        while (!$publishSuccess)
+#        while (!$publishSuccess)
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip retries, if it is false, then keep the current logic of asking the user for input.